VERSION

```diff
@@ -0,0 +1 @@
+v0.0.1
```

Comment:
shouldn't we start in the v0.1.0 version? meaning the first set of features exported here?